packages/lib/passport-interface/src/Credential.ts

```diff
@@ -82,14 +82,22 @@ export class VerificationError extends Error {}
  * of the signature.
  *
  * This function only proves that the credential is formally valid. It does
- * *not* check if the EmailPCD was signed by Zupass. In both IssuanceService
- * and in CredentialSubservice (part of Podbox), this additional check is
- * performed, because those servies have higher context on what might be a
- * valid signing key (e.g. one specified in an environment variable, which
+ * *not* check if the EmailPCD was signed by Zupass. Callers which depend on
+ * the email address should perform this check using the optional
+ * `isTrustedEmailPCDSigner` argument, or by checking the `emailPCDSigner`
+ * field in the result.  Callers know what might be a valid signing key for
+ * their use case (e.g. one specified in an environment variable, which
  * application code has access to but library code such as this does not).
+ *
+ * @param credential the credential to verify
+ * @param isTrustedEmailPCDSigner is called with the public key of the
+ *  signer of the email PCD (if included) to validate whether it should
+ *  be trusted
+ * @throws VerificationError if the credential is invalid
  */
 export async function verifyCredential(
-  credential: Credential
+  credential: Credential,
+  isTrustedEmailPCDSigner?: (emailPCDSigner: EdDSAPublicKey) => boolean
```

Comment:
A simpler version of this would be to pass an array of keys to accept, which might be more obvious how to use correctly.  The function gives the caller more flexibility, including to throw their own exception (which I did in the CredentialSubservice) or to accept anything.